high_card.rb

```diff
@@ -1,5 +1,57 @@
+require "pry"
 # Use these two arrays to generate a deck of cards.
-ranks = ["A", 2, 3, 4, 5, 6, 7, 8, 9, 10, "J", "Q", "K" ]
+classes = [2, 3, 4, 5, 6, 7, 8, 9, 10, "J", "Q", "K", "A" ]
 suits = [ "hearts", "spades", "clubs", "diamonds" ]
+$players = []
+p_1 = $players[0]
+p_2 = $players[1]
+p_3 = $players[2]
 
-players = []
+def build_deck(classes, suits)
+  # build a deck
+    # initialize an empty deck
+    deck = []
+    # loop through ranks
+      # .each with index -> to pull the worth of a card
+        Classes.each_with_index do |classes, index|
```

Comment:
Aslo, when you are enumerating, you will get one "class" at a time, not "classes" each time.  I recommend you avoid possible reserved words, like Class.